Mathlib/Data/Fintype/Sum.lean

```diff
@@ -139,10 +139,11 @@ theorem Fintype.card_subtype_or_disjoint (p q : α → Prop) (h : Disjoint p q)
 
 section
```

Comment:
I am only noticing this because of the PR: should this `section`/`end` pair be removed?